index.js

```diff
@@ -185,25 +185,25 @@ module.exports = {
     "no-extra-parens": 0,
     "no-underscore-dangle": 0,
     "one-var": 0,
-    "padded-blocks": [ 2, "never" ],
-    "semi": [ 2, "never" ],
-    "semi-spacing": [ 2, {
+    "padded-blocks": [2, "never"],
+    "semi": [2, "always"],
+    "semi-spacing": [2, {
       "before": false,
       "after": true
-    } ],
-    "space-before-keywords": [ 2, "always" ],
+    }],
+    "space-before-keywords": [2, "always"],
     "space-after-keywords": 2,
-    "space-before-function-paren": [ 2, {
+    "space-before-function-paren": [2, {
       "anonymous": "always",
-      "named": "always"
-    } ],
+      "named": "never"
+    }],
     "space-infix-ops": 2,
     "space-return-throw-case": 2,
     "spaced-comment": 2,
     "no-multi-spaces": 0,
-    "max-len": [ 2, 100, 2, {
+    "max-len": [2, 100, 2, {
```

Comment:
I very seriously doubt we are going to reopen the discussion about our style guide because someone decided they didn't like the current line length specification.